app/presenters/notifications/setup/download-recipients.presenter.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+/**
+ * Formats data for the `/notifications/setup/download` link
+ * @module DownloadRecipientsPresenter
+ */
+
+const { contactName } = require('../../crm.presenter.js')
+const { formatDateObjectToISO } = require('../../../lib/dates.lib.js')
+const { transformArrayToCSVRow } = require('../../../lib/transform-to-csv.lib.js')
+
+/**
+ * Formats data for the `/notifications/setup/download` link.
+ *
+ * This function takes an array of recipient objects and transforms it into a CSV
+ * string suitable for download.
+ *
+ * The headers are fixed and in the correct order. If a value for a row does not match the header then it will default
+ * to an empty string.
+ *
+ * @param {object[]} recipients - An array of recipients
+ *
+ * @returns {string} - A CSV-formatted string that includes the recipients' data, with the first
+ * row as column headers and subsequent rows corresponding to the recipient details.
+ */
+function go(recipients) {
+  const formattedData = _transformForCsv(recipients)
+  return _transformJsonToCsv(formattedData)
+}
+
+/**
+ * Transforms the recipients' data into a CSV-compatible format.
+ *
+ * The order of the object dictates the CSV header order.
+ *
+ * @private
+ */
+function _transformForCsv(recipients) {
+  return recipients.map((recipient) => {
+    const { contact } = recipient
+
+    return {
+      Licences: recipient.licence_ref,
+      'Return references': recipient.return_reference,
+      'Returns period start date': formatDateObjectToISO(recipient.start_date),
+      'Returns period end date': formatDateObjectToISO(recipient.end_date),
+      'Returns due date': formatDateObjectToISO(recipient.due_date),
+      'Message type': contact ? 'letter' : 'email',
+      'Message reference': 'invitations',
+      'Licence holder': contact ? contactName(recipient.contact) : '',
+      'Recipient name': '',
+      Email: recipient.email || '',
+      'Address line 1': contact ? contact.addressLine1 : '',
```

Comment:
I was going to lift this out into a seperate function for the addresses. But we loose the contaxt of the header order.